src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -206,23 +206,33 @@ Type-specific fields:
 
 Note that unlike `object`, there is no `nullable` field on `params`.
 
+`params` can not be declared as top-level named types in a schema `defs` array. This means that they can not be reference by a `ref` or `union`.
```

Comment:
also i know `params` is technically plural, but this reads weird to me and when referred to as a type, I think we should treat it has singular. for instance: "`params`  can not be declared as a top-level named type..."